ansible/library/show_interface.py

```diff
@@ -65,9 +73,11 @@ class ShowInterfaceModule(object):
     def __init__(self):
         self.module = AnsibleModule(
             argument_spec=dict(
-            command=dict(required=True, type='str'),
-            interfaces=dict(required=False, type='list', default=None),
-            up_ports=dict(type='raw', default={}),
+                command=dict(required=True, type='str'),
+                namespace=dict(required=False, type='str', default=None),
+                interfaces=dict(required=False, type='list', default=None),
+                up_ports=dict(type='raw', default={}),
+                include_internal_intfs=dict(required=False, type=bool, default=False),
```

Comment:
not a big deal, but you can probably set this variable to True by default since it is only impacting multi-ASIC with fabric ASIC. Unless by default we don't want to see them.

Reply:
@yxieca, I wanted the module to have the same behavior as the CLI command where to dump the internal information we need extra parameter.